File: labs/LAB2/Anagram.py
class Anagram:

    def isAnagram(s,t):
        if len(s) != len(t):
            return False
        else:
            s = s.lower()
            t = t.lower()

            sArray = [char for char in s]
            tArray = [char for char in t]

            sArray.sort()
            tArray.sort()

            for i in range(len(sArray)):
                if sArray[i] != tArray[i]: return False
        return True
    
s = "dog"
t = "gcd"

print(Anagram.isAnagram(s,t))
        

File: labs/LAB2/Anagram.java
package labs.LAB2;
import java.util.*;

public class Anagram {

    public static boolean isAnagram(String s, String t) {
        
        if(s.length() != t.length()){
            return false;
        }else{
            s = s.toLowerCase();
            t = t.toLowerCase();

            char sArray[] = s.toCharArray();
            char tArray[] = t.toCharArray();

            Arrays.sort(sArray);
            Arrays.sort(tArray);

            for(int i = 0; i < sArray.length; i++){
                if(sArray[i] != tArray[i]){
                    return false;
                }
            }
        }
        return true;
    }

    public static void main(String[] args) {
        String s = "rtvxv";
        String t = "xvrtA";

        System.out.println(isAnagram(s,t));
    }
}
